Alliance.Common/Extensions/UsableEntity/NetworkMessages/FromServer/RemoveEntity.cs

```diff
@@ -0,0 +1,42 @@
+﻿using TaleWorlds.Library;
+using TaleWorlds.MountAndBlade;
+using TaleWorlds.MountAndBlade.Network.Messages;
+
+namespace Alliance.Common.Extensions.UsableEntity.NetworkMessages.FromServer
+{
+    [DefineGameNetworkMessageTypeForMod(GameNetworkMessageSendType.FromServer)]
+    public sealed class RemoveEntity : GameNetworkMessage
+    {
+        public Vec3 Position { get; private set; }
+
+        public RemoveEntity()
+        {
+        }
```

Comment:
Useful ?

Reply:
Yep. When scanning the assemblies for NetworkMessages, the engine looks for classes with the sealed keyword and an empty constructor.

Reply:
Ok. Thank for explanation.
 A comment upper this explaining that could then be useful :)